Failed watching pipe status -l

<!--- Provide a general summary of the issue in the Title above -->

**Description**
```bash
» glab pipe status -l                                  
(running) • less than a minute ago      build           build

https://gitlab.......
SHA: ba5954.....
Pipeline State: running
panic: send on closed channel

goroutine 1 [running]:
github.com/gosuri/uilive.(*Writer).Stop(...)
        /home/runner/go/pkg/mod/github.com/gosuri/uilive@v0.0.4/writer.go:119
github.com/profclems/glab/commands/pipeline/status.NewCmdStatus.func1(0xc00042db80, 0xc00033cfe0, 0x0, 0x1, 0x0, 0x0)
        /home/runner/work/glab/glab/commands/pipeline/status/pipeline_status.go:133 +0xb92
github.com/spf13/cobra.(*Command).execute(0xc00042db80, 0xc00033cfd0, 0x1, 0x1, 0xc00042db80, 0xc00033cfd0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc0004e0dc0, 0xc000128050, 0x3, 0x3)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
main.main()
        /home/runner/work/glab/glab/cmd/glab/main.go:114 +0x712

```

**Your Environment**
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used: 1.11.1 
* Operating System and version: macOS 11.0.1 (20B29)

--- a/commands/pipeline/status/pipeline_status.go
+++ b/commands/pipeline/status/pipeline_status.go
@@ -71,6 +71,7 @@ func NewCmdStatus(f *cmdutils.Factory) *cobra.Command {
 
 				// start listening for updates and render
 				writer.Start()
+				defer writer.Stop()
 				for isRunning {
 					jobs, err := api.GetPipelineJobs(apiClient, runningPipeline.ID, repo.FullName())
 					if err != nil {
@@ -127,7 +128,6 @@ func NewCmdStatus(f *cmdutils.Factory) *cobra.Command {
 							isRunning = false
 						}
 					}
-					writer.Stop()
 
 					if retry == "View Logs" {
 						// ToDo: bad idea to call another sub-command. should be fixed to avoid cyclo imports

